glide.yaml

```diff
@@ -0,0 +1,12 @@
+package: github.com/Clever/kinesis-alerts-consumer
+import:
+- package: github.com/Clever/amazon-kinesis-client-go
+  subpackages:
+  - batchconsumer
+- package: gopkg.in/Clever/kayvee-go.v6
+  subpackages:
+  - logger
+- package: github.com/signalfx/golib
+- package: github.com/Clever/syslogparser
```

Comment:
Hmm, `syslogparser` should be an indirect dependency, no?  Since it's amazon-kinesis-client-go that does all the syslog parsing?